src/server/object_services/object_server.js

```diff
@@ -330,9 +341,32 @@ async function create_multipart(req) {
         multipart_id: multipart._id,
         chunk_split_config: req.bucket.tiering.chunk_split_config,
         chunk_coder_config: tier.chunk_config.chunk_coder_config,
+        encryption: obj.encryption
     };
 }
 
+// function _upload_part_encryption_validation(src_enc, req_enc) {
```

Comment:
This function was removed since it basically is the same as ```_check_encryption_permissions```.
This is needed and cannot be performed on the endpoint since it needs the MD of the object.